help/status-and-availability.md

```diff
@@ -75,6 +75,22 @@ and compose box. Status emoji and status messages are also shown on
 
 {end_tabs}
 
+### Configure how a user status is displayed in the right sidebar
```

Comment:
Maybe make the heading a bit more succinct -- something like this?

### Configure how statuses are displayed

You can choose whether or not status text is displayed in the left sidebar. With the **compact** option, only status emoji are shown.